### PR TITLE
IE11 ES6 fixes

### PR DESCRIPTION
**Pull request by RayNjeri**
Signed-off-by: Rachael Njeri <rachael.njeri@andela.com>

Resolves [NO ISSUE] but relates to #2016 

Follows on from #2016

Overall change: Reverts some es6 linting to ES5

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-locales/moment/fa.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ const jalaliFormats = ['D MMMM YYYY', 'LL'];
 
 moment().locale('fa');
 moment.updateLocale('fa', {
-  postformat(string) {
+  // eslint-disable-next-line object-shorthand
+  postforma: function(string) {
```
Comment (rhenshaw56) [inline packages/utilities/psammead-locales/moment/fa.js]: missing 't'

**Review by rhenshaw56 — CHANGES_REQUESTED** [review]

File: packages/utilities/psammead-locales/moment/fa.js (hunk 1 above)
Comment (RayNjeri) [reply packages/utilities/psammead-locales/moment/fa.js]: thanks

**Review by dr3 — COMMENTED** [review]
LGTM, but is it possible to just make eslint global lint `/moment` to es5? would be easier to catch stuff in future

**Review by jamesdonoh — APPROVED** [review]
Good catch, thanks.

**Comment by jamesdonoh** [comment]
@dr3 Incoming issue to improve linting which we'll do immediately after

**Comment by RayNjeri** [comment]
@jamesdonoh i can just add the fix that @dr3 suggested here instead of creating an issue

**Review by dr3 — APPROVED** [review]
LGTM untill we add the es5 linting :)

**Review by rhenshaw56 — APPROVED** [review]

**Comment by jamesdonoh** [comment]
Linting improvements: https://github.com/bbc/psammead/issues/2025

**Pull request merged**